src/js/structs/FormAction.js

```diff
@@ -0,0 +1,21 @@
+class FormAction {
+  constructor(path, value, type = 'SET') {
+    Object.defineProperties(this, {
```

Comment:
I still plan to add a constant file for all the types. With those we could essentially do a simple validation. I just wanted to do that in a later step, as this PR is already introducing a couple of new things. 

So do you think I should introduce this also in this PR or rather in a later one?